### ISSUE TITLE
Request for Grade

### ISSUE THREAD

**Issue by VanshitaGandhi** [VanshitaGandhi/web-design-2-independent-project#10]
@hellachella 

I believe I would give myself an A+

I applied all the concepts we learnt this term like using flexbox, grids, media queries, ensuring accessibility, exporting images from illustrator and photoshop and smushing them properly, using svg icons, and performing browser, accessibility and performance testing. The content of the website is well thought out and includes all the important aspects that a website should have. The grammar has been properly double-checked to avoid any errors. I chose deep hues in contrast to rich bright ones to keep the website clean to avoid any chaos and confusion.
The main challenge that I faced was using grids, but I overcame that by referring to the course material provided. I committed and pushed the code regularly. In addition, I completed the initial steps of the project that included sketches and layouts in a timely and disciplined manner.
